### DISCUSSION TITLE
[PSD1] Exercise 1.45

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#120] [Esercizi (A-L)]
**Exercise 1.45**

A particular DSL modem operates at 768 kbits/sec. How many bytes can it receive in 1 minute?

**Comment by Elia-Belli** [comment]
- 768 kbits/s = 96 kbytes/s
- Se in 1s possiamo ricevere 96000 bytes in 1min ne riceviamo 5760000, ovvero 5.76 MB
